setup.sh

```diff
@@ -7,7 +7,12 @@ mkdir -p "$INSTALL_DIR"
 git clone --depth 1 https://gerrit.wikimedia.org/r/mediawiki/core "$INSTALL_DIR"/mediawiki
 git clone --depth 1 https://gerrit.wikimedia.org/r/mediawiki/skins/Vector "$INSTALL_DIR"/mediawiki/skins/Vector
 
-cd "$INSTALL_DIR"/mediawiki
+
+DOCKER_MW_PATH="$INSTALL_DIR"/mediawiki
+
+echo "DOCKER_MW_PATH=$DOCKER_MW_PATH" >> local.env
```

Comment:
Might be nice to not do this if the .env file already exists? maybe?
But this is also on first run only, so maybe is fine...

Reply:
I think this probably makes sense to always do because pointing the dev environment to the code it just cloned is part of setup

Reply:
I don't know how Docker reads the `.env` file exactly, but if it reads it similar to how bash would read assignments, then this might need escaping/quotation in order to perist as intended otherwise the meaning of spaces or quotes would be lost.

```
DOCKER_MW_PATH=${DOCKER_MW_PATH@Q}`
```

That might do the trick